### ISSUE TITLE
(Accessibility) With "High Contrast - White" theme, some error boxes are difficult for colorblind users to see

### ISSUE THREAD

**Issue by bwateratmsft** [microsoft/AzureStorageExplorer#5631]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.23.0

### Architecture

x64

### Storage Explorer Build Number

20220223.14

### Platform

Windows

### OS Version

Windows 11

### Bug Description

With the "High Contrast - White" theme, certain error box highlights are hard to see (screenshot below will explain better)

### Steps to Reproduce

1. Turn on "High Contrast - White" theme
2. Edit blob properties
3. Add three Metadata tags
4. Set one of the tags to be a duplicate name of the other

### Actual Experience

![image](https://user-images.githubusercontent.com/36966225/163430608-cf523487-5c9e-43db-8270-bc3dc46102a3.png)

For colorblind users, it can be difficult to distinguish the thin red outline of the top two boxes from the black outline of the lower box. I spoke to a colorblind user with deuteranomaly, who said that it would be easier to distinguish if the red outline was thicker.

**Comment by craxal** [comment]
Speak to some accessibility experts to see if there's a standard way to handle this.
